content/en/docs/tasks/observability/distributed-tracing/configurability/index.md

```diff
@@ -53,6 +53,41 @@ customization. In particular, make sure that the tracing backend address is
 always provided in the annotations to ensure that the traces are reported
 correctly for the workload.
 
+## Installation
+
+Using these features opens new possibilities for managing traces in your environment.
+
+In this example, we will sample all traces and add a tag named `service.istio.io/canonical-name`
```

Comment:
Can we use a different example?  After https://github.com/istio/istio/pull/28801, this information should already be added to trace spans?

Reply:
I added `ISTIO_META_CLUSTER_ID`. Tell me if you think it makes sense. I don't have to specify an environmental variable and this may be handy for those in multicluster environments.